android/app/src/main/java/mqttController/CarConnect.java

```diff
@@ -131,6 +131,22 @@ public void publish(String topic, String message, int qos, IMqttActionListener p
         mMqttClient.publish(topic, message, qos, publishCallback);
     }
 
+    public void successFeedbackMessage(){
+        Toast toast = Toast.makeText(context, SUCCESSFUL_CONNECTION, Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.TOP,0,0);
+        toast.show();
+    }
 
+    public void failedFeedbackMessage(){
+        Toast toast = Toast.makeText(context, FAILED_CONNECTION, Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.TOP,0,0);
+        toast.show();
+    }
+
+    public void lostFeedbackMessage(){
+        Toast toast = Toast.makeText(context, LOST_CONNECTION, Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.TOP,0,0);
+        toast.show();
+    }
```

Comment:
I think that this code could be reduced to one method only if there was one feedbackMessage() method with a parameter that's either a SUCCESSFUL_CONNECTION, FAILED_CONNECTION, or LOST_CONNECTION depending on where the method is called.

Reply:
Really good idea! This helps a lot with the readability of code 👍